manifests/0000_70_dns-operator_00-custom-resource-definition.yaml

```diff
@@ -68,6 +428,23 @@ spec:
                     type: string
                   type:
                     type: string
+                required:
+                - type
+                - status
                 type: object
               type: array
+          required:
+          - clusterIP
+          - clusterDomain
           type: object
+      type: object
+  versions:
+  - name: v1
+    served: true
+    storage: true
+status:
```

Comment:
Seems like the `status:` stanza should be omitted, but it shouldn't hurt anything.